eks/modules/services/scfcharts.tf

```diff
@@ -185,29 +179,35 @@ resource "null_resource" "wait_for_uaa" {
   }
 }
 
-provider "helm" {
-  alias = "metrics"
-  version = "1.0.0"
+resource "kubernetes_namespace" "metrics" {
+  depends_on = [
+    kubernetes_config_map.aws_auth
+  ]
 
-  kubernetes {
-    config_path = var.kubeconfig_file_path
+  metadata {
+    name = "metrics"
+  }
+
+  timeouts {
+    delete = "30m"
   }
 }
 
 resource "helm_release" "metrics" {
   depends_on = [
     helm_release.stratos,
     kubernetes_namespace.metrics,
-    null_resource.wait_for_uaa
+    null_resource.wait_for_uaa,
+    null_resource.cluster_issuer_setup,
+    local_file.kubeconfig_file
   ]
 
-  provider   = helm.metrics
   name       = "susecf-metrics"
   repository = "https://kubernetes-charts.suse.com"
   chart      = "metrics"
   version    = "1.2.1"
   namespace  = "metrics"
-  wait       = "false"
+  wait       = "true"
```

Comment:
We don't need this to be true - there's nothing else dependent on it. The CAP deployment will take its time too and we are telling the end user to check the pod status for readiness.